map2annotation/fetch_map2anno_tools.sh

```diff
@@ -1,17 +1,10 @@
 #!/bin/bash
 MAP2ANNO_DIR=$(cd $(dirname "$0") && pwd)
 
-##### build hacked checkstyle
```

Comment:
The `hacked checkstyle` works as a back-end to `multiDeclRefactor`.  `hacked checkstyle` detects multi-declarations problems and send diagnostic reports to `multiDeclRefactor`, and then `multiDeclRefactor` do refactors based on the reports.

So in this PR, I changed `hacked checkstyle` as a dependency to `multiDeclRefactor`, and move `checkstyle` inside to `multiDeclRefactor`.